packages/@aws-cdk-testing/framework-integ/test/aws-cloudwatch/test/integ.alarm-and-dashboard.ts

```diff
@@ -61,6 +61,8 @@ dashboard.addWidgets(new cloudwatch.SingleValueWidget({
 }));
 dashboard.addWidgets(new cloudwatch.LogQueryWidget({
   title: 'Errors in my log group',
+  // Setting an account ID just for this LogQueryWidget
+  accountId: '123456789',
```

Comment:
we have specific accountIds that won't get flagged by security. is this one of them? also, account ids are 12 numbers long

Reply:
Oh that's good to know - I just put in a random number here, will update it 👍